setup.py

```diff
@@ -327,7 +327,7 @@ def make_release_tree(self, base_dir, files):
         "numpy>=1.21",
         "packaging>=20.0",
         "pillow>=6.2.0",
-        "pyparsing>=2.3.1",
+        "pyparsing>=2.3.1,<3.1",
```

Comment:
```suggestion
        "pyparsing>=2.3.1",
```

Reply:
I think I agree here, let's fix CI first and if upon further investigation we determine that the fix is upstream we can do a more narrow `!=3.1.0` once that is pushed up rather than a full `<` pin